### DISCUSSION TITLE
official flatpak support

### DISCUSSION THREAD

**Discussion by 00pandora00** [VSCodium/vscodium#1748] [Ideas]
hi there is any plan for official flatpak support? i think i will help a lot if the official team contributes to the flatpak version, and this can also help reduce maintenance on Linux.

**Comment by daiyam** [comment]
flatpak is available here https://github.com/flathub/com.vscodium.codium

What the issue with that?

**Comment by 00pandora00** [comment]
> flatpak is available here https://github.com/flathub/com.vscodium.codium
> 
> What the issue with that?

hi, the problem is i don't know if the flatpak is official maintained for the team of vscodium  (i don't see the check mark) 
if the flatpak is official supported: 
i will change this issue to "verifice the app in flathub". With that, the vscodium team will avoid misunderstandings like mine (I thought vscodium flatpak was not supported by the vscodium team)
if the flatpak is not official supported:
"pls, read again my first message"
